### PR TITLE
Use the same ports in binary, 0 in tests

### LINKED ISSUE #269: Make test config 7 verifiers 3 operators
### Issue Description

For better understand the real world case

### Tasks

_No response_

### Deadline of the Issue

-

### PR DESCRIPTION
**Pull request by ekrembal**
Closes #269

### CODE REVIEW

**Review by ozankaymak — DISMISSED** [review]

File: core/src/servers.rs [hunk 1]
```diff
@@ -111,6 +111,11 @@ pub async fn create_aggregator_server(
     Ok((client, handle, addr))
 }
 
+fn is_test_env() -> bool {
+    // if thread name is not main then it is a test
+    thread::current().name().unwrap_or_default() != "main"
```
Comment (ceyhunsen) [inline core/src/servers.rs:116]: Maybe use `if cfg!()`?

```rust
if cfg!(test) {
    true
} else {
   false
}
```

Comment (ekrembal) [reply core/src/servers.rs:116]: it didn't work

Comment (ceyhunsen) [reply core/src/servers.rs:116]: Does this also not work?

```rust
#[cfg(test)]
true
#[cfg(not(test)]
false
```

Comment (ceyhunsen) [reply core/src/servers.rs:116]: Reported as not working again

**Review by ozankaymak — APPROVED** [review]

**Review by ceyhunsen — DISMISSED** [review]

**Pull request merged**